File: api_example01/TodoApi/Models/TodoItem.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TodoApi.Models;

[Table("to_do")]
public class TodoItem
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    [Column("id")]
    public int Id { get; set; }

    [Column("deleted")]
    public bool Deleted { get; set; } = false;

    [Column("description")]
    [MaxLength(255)]
    public string? Description { get; set; }

    [Column("task")]
    [MaxLength(255)]
    public string? Task { get; set; }
}

File: api_example01/TodoApi_Springboot/src/main/java/com/example/todoapi/model/TodoItem.java
package com.example.todoapi.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "to_do")
public class TodoItem {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Integer id;

    @Column(name = "deleted")
    private boolean deleted = false;

    @Column(name = "description", length = 255)
    private String description;

    @Column(name = "task", length = 255)
    private String task;
}
